dbinit/liquibase.gradle

```diff
@@ -0,0 +1,78 @@
+configurations {
+  liquibase
+}
+
+dependencies {
+  liquibase("org.liquibase.ext:liquibase-hibernate4:3.6")
+}
+
+//loading properties file.
+Properties liquibaseProps = new Properties()
+liquibaseProps.load(new FileInputStream("dbinit/src/main/liquibase/liquibase-task.properties"))
+
+Properties applicationProps = new Properties()
+applicationProps.load(new FileInputStream("core/src/main/resources/application.properties"))
+
+String databaseUrl = applicationProps.getProperty('spring.datasource.url')
+String hibernateUrl = "hibernate:spring:" + liquibaseProps.getProperty('liquibase.domain.package') + "?dialect=" + applicationProps.getProperty('spring.jpa.database-platform') + "&hibernate.ejb.naming_strategy=" + applicationProps.getProperty('spring.jpa.hibernate.naming_strategy')
+String userName = applicationProps.getProperty('spring.datasource.username')
+String password = applicationProps.getProperty('spring.datasource.password')
+
+String changeLogPath = liquibaseProps.getProperty('liquibase.changelog.path')
+
+task liquibaseGenerateChangeLogFromCode(type: JavaExec) {
```

Comment:
I will create a wiki page to describe the process around Liquibase, including the purpose of these tasks. I will do that along with the first schema update.